docs/install.md

```diff
@@ -1,5 +1,7 @@
 # Installation
 
+If PHP is not installed on your environment, install it now; PHP is a requirement for Dock CLI.
```

Comment:
Good point. But as these versions are not maintained anymore, nobody should use them anyway. To summarise, I agree either way :)